amundsen_application/static/js/components/common/ResourceListItem/TableListItem/tests/index.spec.tsx

```diff
@@ -49,22 +49,23 @@ describe('TableListItem', () => {
         });
 
         it('renders secondary description w/ getDateLabel value if table has last_updated_epoch ', () => {
-          subject.instance().getDateLabel = jest.fn(() => 'Mar 28, 2019')
+          subject.instance().getDateLabel = jest.fn(() => 'Mar 28, 2019');
           props.table.last_updated_epoch = 1553829681;
           subject.setProps(props);
           expect(subject.find('.content').children().at(1).children().at(1).text()).toEqual('Mar 28, 2019');
         });
     });
 
+    /*
+    Note: Jest will convert date to UTC, expect to see different strings for an epoch value in the tests vs UI
```

Comment:
As discussed offline, I'll look into this. Feel free to comment out any tests involving date/time that block development.